engine/runtime/src/main/java/org/enso/interpreter/service/error/ExitException.java

```diff
@@ -0,0 +1,63 @@
+package org.enso.interpreter.service.error;
+
+import com.oracle.truffle.api.TruffleStackTraceElement;
+import com.oracle.truffle.api.exception.AbstractTruffleException;
+import com.oracle.truffle.api.interop.ExceptionType;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.library.ExportLibrary;
+import com.oracle.truffle.api.library.ExportMessage;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
+
+/** Thrown when {@code System.exit} call was called during execution. */
+@ExportLibrary(InteropLibrary.class)
+public class ExitException extends AbstractTruffleException implements ServiceException {
+  private final int exitCode;
+
+  public ExitException(int code, Node location) {
```

Comment:
Can we move this class into the same package that throws it and make constructor (or rather whole class) package private? Nobody else outside of the `engine/runtime` shall be accessing this exception, I hope.

Reply:
Made package-private in 78373a20b0e71ce04a252238d9a8d01fd44c3b5f